src/bundle/Resources/views/content/tabs/versions.html.twig

```diff
@@ -0,0 +1,18 @@
+<div class="ez-versions-list-container">
+    {%  if draftVersions %}
+        <h2 class="ez-list-toolbar-label">{{'locationview.versions.draft.under.edit'|trans({}, 'locationview')}}</h2>
+        {% include '@EzSystemsHybridPlatformUi/content/tabs/versions/table.html.twig' with {'versions': draftVersions} %}
+
+    {% endif %}
+
+    {%  if publishedVersions %}
+        <h2 class="ez-list-toolbar-label">{{'locationview.versions.published.version'|trans({}, 'locationview')}}</h2>
+        {% include '@EzSystemsHybridPlatformUi/content/tabs/versions/table.html.twig' with {'versions': publishedVersions} %}
+    {% endif %}
+
+    {%  if archivedVersions %}
+        <h2 class="ez-list-toolbar-label">{{'locationview.versions.archived.versions'|trans({}, 'locationview')}}</h2>
```

Comment:
ezplatform-i18n is read-only, it gets the sources from each individual package.
We must start adding translation sources to this repository, and sync the files to ezplatform-i18n.

About translation strings, we use JmsTranslationBundle, that provides |desc twig function. It is used to provide the default translation while using a static identifier. It would be good to add the translations straight away.

Example:
```
{{'locationview.versions.archived.versions'|trans({}, 'locationview')|desc('Archived versions')}}
```

You can find examples in the [kernel](https://github.com/ezsystems/ezpublish-kernel/blob/master/eZ/Bundle/EzPublishCoreBundle/Resources/views/fielddefinition_settings.html.twig#L18).

I'll create a story to configure translations extraction in this repository.